File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.c
#include <stdio.h>

int main()
{
    int n, students[30] = {0};
    
    for (int i = 0; i < 28; i++) {
        scanf("%d", &n);
        
        students[n - 1]++;
    }
    
    for (int i = 0; i < 30; i++) if (students[i] == 0) printf("%d\n", i + 1);
}

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    
	    int[] students = new int[30];
	    
	    for (int i = 0; i < 28; i++) {
	        int n = sc.nextInt();
	        
	        students[n - 1]++;
	    }
	    
	    for (int i = 0; i < 30; i++) if (students[i] == 0) System.out.println(i + 1);
	}
}